### PR TITLE
adding flux v2 overview

### PR DESCRIPTION
**Pull request by adusumillipraveen**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: source/ways-of-working/new-component/gitops-flux.html.md.erb [hunk 1]
```diff
@@ -17,30 +17,54 @@ One of our principles being <b>Everything as Code</b>, we practise [Gitops](http
 
 [Flux](https://docs.fluxcd.io/) is a tool that automatically ensures that the state of a cluster matches the config in [Git](https://github.com/hmcts/cnp-flux-config). 
 
-- It is an operator in the cluster to trigger deployments inside Kubernetes, which means you don't need a separate CD tool.
-- It monitors all relevant image repositories, detects new images, triggers deployments and updates the desired running configuration. 
+- Flux uses Gitops Toolkit, a set of APIs and controllers that make up the runtime for Flux.
```
Comment (timja) [inline source/ways-of-working/new-component/gitops-flux.html.md.erb:20]: can we archive this page with a disclaimer at the top? SDS hasn't started migration and we share docs.

File: source/ways-of-working/new-component/gitops-flux.html.md.erb [hunk 2]
```diff
@@ -12,6 +12,7 @@ One of our principles being <b>Everything as Code</b>, we practise [Gitops](http
 - Git as single source of truth.
 - Declarative Infrastructure and Application deployment through pull requests.
 
+> If you are looking for Flux v1 documentation, please see [V1 docs](flux-v1.html)
```
Comment (timja) [inline source/ways-of-working/new-component/gitops-flux.html.md.erb:15]:
```suggestion
> If you are looking for Flux v1 documentation (SDS is still on this version), please see [V1 docs](flux-v1.html)
```

**Review by timja — APPROVED** [review]
lgtm, I assume you've checked all rendering looks fine?

File: source/ways-of-working/new-component/gitops-flux.html.md.erb (hunk 2 above)
Comment (adusumillipraveen) [reply source/ways-of-working/new-component/gitops-flux.html.md.erb:15]:
```suggestion
> Please note SDS and [few clusters of CFT](https://github.com/hmcts/cnp-flux-config#migration-status) are still on v1, refer [V1 docs](flux-v1.html). 
```

**Pull request merged**